apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.ts

```diff
@@ -61,6 +61,8 @@ export class InvoicePreviewDetailsComponent implements OnInit, OnChanges, AfterV
     @Output() public sendEmail: EventEmitter<string | { email: string, invoiceType: string[], invoiceNumber: string }> = new EventEmitter<string | { email: string, invoiceType: string[], invoiceNumber: string }>();
     @Output() public processPaymentEvent: EventEmitter<InvoicePaymentRequest> = new EventEmitter();
     @Output() public refreshDataAfterVoucherUpdate: EventEmitter<boolean> = new EventEmitter();
+    @Output() public isOpenAdvanceReceiptModal: EventEmitter<boolean> = new EventEmitter();
```

Comment:
Consider starting name of the event emitter with **on** for eg: `onAdvanceReceiptModalOpen`, avoid starting handler name of event with **on** however.
Add documentation to new member declaration